### PR TITLE
Feat(eos_designs): Add `is_deployed` & `mgmt_interface_description` to schema

### PR DESCRIPTION
**Pull request by chetryan**
## Change Summary

Add some of the schema keys that were missing as identified in #2834 

## Related Issue(s)

Related to  #2834 

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes

- Define `is_deployed` in the `eos_designs` schema.
- Define `mgmt_interface_management` in the `eos_designs` schema.

## How to test

- run `molecule test -s eos_designs_unit_tests`

### User Checklist

- Review whether updates to documentation are enough/correct.
- Review whether schema needs further/stricter validation.

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/is_deployed.schema.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  is_deployed:
+    description: |
+      Is device already deployed in the fabric. If this is not defined, it is
+      assumed device is deployed. When set to false, interfaces toward this
+      device may be shutdown and `eos_config_deploy_cvp` or
+      `eos_config_deploy_eapi` do not try to
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/is_deployed.schema.yml]:
```suggestion
      device may be shutdown and `eos_config_deploy_cvp` does not try to
```

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/mgmt_interface_description.schema.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  mgmt_interface_description:
+    type: str
+    description: |
+      Description on the Management interface to override
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/mgmt_interface_description.schema.yml]:
```suggestion
      Management interface description
```

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/mgmt_interface_description.schema.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+      the default `oob_management`.
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/mgmt_interface_description.schema.yml]:
```suggestion
    default: oob_management
```

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/is_deployed.schema.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  is_deployed:
+    description: |
+      Is device already deployed in the fabric. If this is not defined, it is
+      assumed device is deployed. When set to false, interfaces toward this
+      device may be shutdown and `eos_config_deploy_cvp` does not try to
+      deploy configuration for this device.
```
Comment (tgodaA) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/is_deployed.schema.yml]: Can we mention about `shutdown_interfaces_towards_undeployed_peers` here.

**Review by tgodaA — APPROVED** [review]
LGTM

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/is_deployed.schema.yml (hunk 4 above)
Comment (chetryan) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/is_deployed.schema.yml]: @ClausHolbechArista  -  Shall I extend the description to mention this?

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/is_deployed.schema.yml]: i will fix

**Review by ClausHolbechArista — APPROVED** [review]

**Pull request merged**